### PR TITLE
Update "rule" polish translation

### PR DESCRIPTION
**Pull request by l3l0**
## Details

Update polish translation for "rule" keyword

## Motivation and Context

Missing translation

### CODE REVIEW

File: gherkin/gherkin-languages.json [hunk 1]
```diff
@@ -2566,7 +2566,7 @@
     "name": "Polish",
     "native": "polski",
     "rule": [
-      "Rule"
+      "Zasada"
```
Comment (jakzal) [inline gherkin/gherkin-languages.json:2569]: Wouldn't `Reguła` be more suitable in this context?

Comment (aurelien-reeves) [reply gherkin/gherkin-languages.json:2569]: I don't know polish
But I know that you Gherkin allows to have several words for a same key.

Maybe you may have both translations: "Zasada" and "Reguła"?

Comment (aurelien-reeves) [reply gherkin/gherkin-languages.json:2569]: @l3l0 do you have the possibility to add the second translation as suggested by @jakzal?

**Comment by aurelien-reeves** [comment]
@l3l0  @jakzal  I've made the update. Does it looks good?

**Comment by jakzal** [comment]
@aurelien-reeves looks good to me 👍

**Comment by aslakhellesoy** [comment]
Hi @l3l0,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Pull request merged**